.gitignore

```diff
@@ -8,3 +8,5 @@ ecs-cli/vendor/pkg
 ecs-cli/.vscode/*
 ecs-cli/.idea
 .idea/*
+*.DS_Store
+*.yml
```

Comment:
We don't want to ignore YAML files, we have things like `buildspec.yml` that should be commited. Can you please remove this?

Reply:
ok